src/main/java/uk/gov/hmcts/divorce/divorcecase/model/CaseData.java

```diff
@@ -364,4 +366,24 @@ public void archiveAlternativeServiceApplicationOnCompletion() {
         }
     }
 
+    @SuppressWarnings("PMD")
+    public AlternativeServiceOutcome buildAlternativeServiceOutcome(AlternativeService alternativeService) {
```

Comment:
it looks to me like this method should be on AlternativeService something like alternativeService.getOutcome()

Reply:
Yep - will move it